src/components/external/DisputeManagement/components/DisputesData/DisputeData.tsx

```diff
@@ -114,28 +117,32 @@ export const DisputeData = ({
 
     const statusBoxOptions = useStatusBoxData(statusBoxProps);
 
+    const showContactSupport = !!dispute?.dispute.defensibility && ['ACCEPTABLE', 'DEFENDABLE_EXTERNALLY'].includes(dispute?.dispute.defensibility);
+    const isDefendable = !!dispute?.dispute.defensibility && dispute?.dispute.defensibility === 'DEFENDABLE' && defendAuthorization;
+    const isAcceptable =
+        !!dispute?.dispute.defensibility && ['ACCEPTABLE', 'DEFENDABLE'].includes(dispute?.dispute.defensibility) && acceptAuthorization;
```

Comment:
To reduce verbosity here, consider capturing `dispute?.dispute.defensibility` in a variable.
```suggestion
    const showContactSupport = !!defensibility && ['ACCEPTABLE', 'DEFENDABLE_EXTERNALLY'].includes(defensibility);
    const isDefendable = !!defensibility && defensibility === 'DEFENDABLE' && defendAuthorization;
    const isAcceptable = !!defensibility && ['ACCEPTABLE', 'DEFENDABLE'].includes(defensibility) && acceptAuthorization;
```